catalyst_voices/lib/pages/registration/upload_seedphrase_dialog.dart

```diff
@@ -0,0 +1,32 @@
+import 'dart:convert';
+
+import 'package:catalyst_voices/widgets/modals/voices_upload_file_dialog.dart';
+import 'package:catalyst_voices_localization/catalyst_voices_localization.dart';
+import 'package:catalyst_voices_models/catalyst_voices_models.dart';
+import 'package:collection/collection.dart';
+import 'package:flutter/material.dart';
+
+class UploadSeedphraseDialog {
+  static Future<List<SeedPhraseWord>> show(BuildContext rootContext) async {
+    final file = await VoicesUploadFileDialog.show(
+      rootContext,
```

Comment:
There doesn't seem to be any other context in scope

```suggestion
  static Future<List<SeedPhraseWord>> show(BuildContext context) async {
    final file = await VoicesUploadFileDialog.show(
      context,
```